FRINXio/frinx-frontend
# Implement cli to inventory

Implement UC shell in Device Inventory UI - see https://github.com/FRINXio/frinx-inventory-server/pull/226.
How to test:
- run inventory server on branch that's in the PR mentioned above
- use `.12` machine in proxy server
- open device inventory in the browser
- check menu for "UC shell"
- play around with it
CAUTION: ssh server sometimes returns kinda weird stuff that xterm library can't process (probably). Use the shell, but be patient and slow.
